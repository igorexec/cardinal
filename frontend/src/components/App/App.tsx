import React from 'react';

const app: React.FC = () => {
  return <div>Welcome to Cardinal!</div>;
};

export const App = app;
