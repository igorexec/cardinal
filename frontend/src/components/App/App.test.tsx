import React from 'react';
import {shallow, ShallowWrapper} from 'enzyme';
import {App} from './App';

describe('App', () => {
  let wrapper: ShallowWrapper;

  beforeAll(() => {
    wrapper = shallow(<App />);
  });

  it('matches snapshot without props', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
