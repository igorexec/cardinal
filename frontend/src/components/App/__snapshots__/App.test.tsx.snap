// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot without props 1`] = `
<div>
  Welcome to Cardinal!
</div>
`;
